src/dataflow/lockfile_packages.rs

```diff
@@ -180,4 +179,8 @@ impl<'a> LockfilePackages<'a> {
             .collect();
         missing_packages
     }
+
+    pub fn extend(&mut self, other_packages: LockfilePackages<'a>) {
+        self.packages.extend(other_packages.packages);
+    }
```

Comment:
Seems reasonable to not use the Extend trait here, Extend trait requires implementing IntoIter, so unless we plan iterate these, I think this is good